### PR TITLE
first commit

### PR DESCRIPTION
**Pull request by screamjohnson**

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,19 @@
+Copyright (c) 2004-2014 Fabien Potencier
```
Comment (EduardTrayan) [inline LICENSE]: Please remove this file because we don't need it

File: UPGRADE-2.2.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
+UPGRADE FROM 2.1 to 2.2
```
Comment (EduardTrayan) [inline UPGRADE-2.2.md]: Please remove this file because we don't need it

File: UPGRADE-2.3.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
+UPGRADE FROM 2.2 to 2.3
```
Comment (EduardTrayan) [inline UPGRADE-2.3.md]: Please remove this file because we don't need it

File: UPGRADE-2.4.md [hunk 4]
```diff
@@ -0,0 +1,9 @@
+UPGRADE FROM 2.3 to 2.4
```
Comment (EduardTrayan) [inline UPGRADE-2.4.md]: Please remove this file because we don't need it

File: UPGRADE.md [hunk 5]
```diff
@@ -0,0 +1,268 @@
+Symfony Standard Edition Upgrade
```
Comment (EduardTrayan) [inline UPGRADE.md]: Please remove this file because we don't need it

File: app/.htaccess [hunk 6]
```diff
@@ -0,0 +1,7 @@
+<IfModule mod_authz_core.c>
```
Comment (EduardTrayan) [inline app/.htaccess]: Please remove this file if you don't use `apache` as web server

File: src/.htaccess [hunk 7]
```diff
@@ -0,0 +1,7 @@
+<IfModule mod_authz_core.c>
```
Comment (EduardTrayan) [inline src/.htaccess]: Please remove this file if you don't use `apache` as web server

File: web/.htaccess [hunk 8]
```diff
@@ -0,0 +1,56 @@
+# Use the front controller as index file. It serves as a fallback solution when
```
Comment (EduardTrayan) [inline web/.htaccess]: Please remove this file if you don't use `apache` as web server

File: web/config.php [hunk 9]
```diff
@@ -0,0 +1,124 @@
+<?php
```
Comment (EduardTrayan) [inline web/config.php]: Please remove this file

**Comment by screamjohnson** [comment]
done

**Comment by EduardTrayan** [comment]
:+1:

**Pull request merged**